src/components/Header/Header.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import styled from 'styled-components';
```

Comment:
@gablabelle Any reason for choosing to styled-components?

Reply:
@nyanofthemoon styled-components are amazing! ;-)

- **Automatic critical CSS**: styled-components keeps track of which components are rendered on a page and injects their styles and nothing else, fully automatically. Combined with code splitting, this means your users load the least amount of code necessary.
- **No CSS class policy, no class name bugs**: styled-components generates unique class names for your styles (scoped). You never have to worry about duplication, overlap or misspellings.
- **Easier deletion of CSS**: it can be hard to know whether a class name is used somewhere in your codebase. styled-components makes it obvious, as every bit of styling is tied to a specific component. If the component is unused (which tooling can detect) and gets deleted, all its styles get deleted with it.
- **Simple dynamic styling**: adapting the styling of a component based on its props or a global theme is simple and intuitive without having to manually manage dozens of classes.
- **Painless maintenance**: you never have to hunt across different files to find the styling affecting your component, so maintenance is a piece of cake no matter how big your codebase is.
- **Automatic vendor prefixing**: write your CSS to the current standard and let styled-components handle the rest.
- **Native mobile support**
- **Theming support**
- **Server-side rendering**: styled-components supports concurrent server-side rendering, with stylesheet rehydration. The basic idea is that every time you render your app on the server, you can create a Server StyleSheet and add a provider to your react tree, that accepts styles via a context API. This doesn’t interfere with global styles, such as keyframes or create Global Style and allows you to use styled-components with React DOM’s various SSR APIs.
- **CSS unit and snapshot testing**: Owing to the fact that styled-components are indeed components, unit testing can be run on them. 
- **Sass and polished support**
- You get all of these benefits while still writing the CSS you know and love, just bound to individual components.